app/Http/Requests/Manager/Notification/StoreRequest.php

```diff
@@ -35,7 +36,7 @@ public function rules()
         return [
             'course_id' => ['required', 'integer', 'exists:courses,id,deleted_at,NULL'],
             'title' => ['required', 'string', 'max:50'],
-            'type' => ['required', new NotificationStoreStatusRule],
+            'type' => ['required', new Enum(TypeEnum::class)],
```

Comment:
記法統一したいのですが、次だと不都合ありますかね？

```suggestion
            'type' => ['required', Rule::enum(TypeEnum::class)],
```

Reply:
直したつもりが見落としていました。
以後、気を付けます。